soc_026_rw0_global_gender_gap/soc_026_rw0_global_gender_gap.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul  1 14:22:50 2021
+
+@author: Jason.Winik
+"""
+
+import pandas as pd
+from pandas import DataFrame
+import numpy as np
+from functools import reduce
+import io
+import requests
+import json
+import os
+import sys
+import tabula
+import dotenv
+dotenv.load_dotenv('C:\\Users\\Jason.Winik\\OneDrive - World Resources Institute\\Documents\\GitHub\\cred\\.env')
```

Comment:
We remove any local directories used before uploading our code to Github. We just assume you have loaded in your .env file before you run any of these scripts.
```suggestion
```